_infra/helm/locust/locustfiles/Pipfile

```diff
@@ -0,0 +1,10 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+[packages]
+google-cloud-storage = "*"
+locust = "*"
```

Comment:
The helm chart deployment uses the `locust` image so that doesn't need installing explicitly.
```
  image:
    repository: locustio/locust
    tag: 2.15.1
```
We've added a `Pipfile` for the purposes of local the dev workspace. This isn't intended to replace the helm chart deployment.